.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,13 @@
+<!--
+Commit message:
+
+- Please use Feature / Add / Change / Fix for player-facing changes. E.g.: "Feature: my cool new feature".
+- Please use Feature / Add / Change / Fix followed by "[NewGRF]" or "[Script]" for moddable changes. E.g.: "Feature: [NewGRF] my cool new NewGRF addition".
+- Please use Codechange / Codefix for developer-facing changes. E.g.: "Codefix #1234: validate against nullptr properly".
```

Comment:
Is it meant that here the 'details' do not start with an uppercase letter? Or shouldn't they start with an uppercase letter in the commit messages?

Reply:
No, that is a bug; let me fix :)

(I am so not used to using an uppercase letter there, that I will need to get used to that :) )